core/server/web/site/app.js

```diff
@@ -111,6 +112,15 @@ module.exports = function setupSiteApp(options = {}) {
     // Fetch the frontend client into res.locals
     siteApp.use(shared.middlewares.frontendClient);
 
+    // Set req.member if a cookie is set
+    siteApp.use(members.authenticateMembersToken);
+    siteApp.use(function (err, req, res, next) {
+        if (err.name === 'UnauthorizedError') {
```

Comment:
Cleanup: This is kinda annoying - it's because we're using `express-jwt` rather than our own middleware, we can't let it continue if the token has expired.

As the siteapp would never throw this error before introduction of the authenticateMembersToken middleware - it's fine for now